src/routes/two-factor-authentication/routes.ts

```diff
@@ -0,0 +1,11 @@
+import * as validatePassword from './validate-password';
+import * as validateCode from './validate-code';
```

Comment:
can you rename `validateCode` to `enable` and put the `code` in the body instead of the query string ?